### ISSUE TITLE
Assignment 12 - Lightning Code

### ISSUE THREAD

**Issue by nviraj** [nviraj/era-v1#46]
Tasks

- [x] change parameters to local
- [x] replace model with week 10
- [x] install lightning and gradio
- [x] fix imports
- [x] change model to lightning module
- [x] loss and accuracy
- [x] optimizer
- [x] training step
- [x] testing step
- [x] dataloader
- [x] scheduler 
- [x] add check-pointing
- [x] add trainer
  - [x] gpu
  - [x] callbacks
- [x] logging
- [x] Solve dataloader issue
- [x] Fix results
- [x] move tensorboard to end
- [x] Fix misclassified images
- [x] grad cam
- [x] remove redundant code in
  - [x] notebook
  - [x] other modules
- [x] test run on local machine
- [x] change parameters to colab dev
- [x] train
- [x] tune
- [x] note final accuracy at top of notebook
- [x] change back parameters to colab prod
